### PR TITLE
Update terms_of_service.md

### PR DESCRIPTION
**Pull request by peterloane**
Edits and typo correction proposals

### CODE REVIEW

File: pages/terms_of_service.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-Last Revised: April 2019
+Last Revised: September 2021
 
 ## Overview
 
 Exercism (also referred to as exercism.io) is an online platform designed to help you improve your coding skills through practice and mentorship.
 
-Our platform comprises of a web site is located at https://exercism.io (and associated subdomains) and a command line application (CLI).
+Our platform comprises of a web site located at https://exercism.io (and associated subdomains) and a command line application (CLI).
```
Comment (IsaacG) [inline pages/terms_of_service.md:9]: It's now exercism.org

Comment (peterloane) [reply pages/terms_of_service.md]: Thanks. I have updated the request.

File: pages/terms_of_service.md [hunk 2]
```diff
@@ -32,22 +32,22 @@ operational, but we cannot offer any guarantees of this.
...
 
 Any clause headings, numbers, and/or emphasis in these Terms are for ease of reference only and shall not be taken into account in the construction or interpretation of the clauses to which they refer.
 
 If any provision of these Terms is found to be illegal, void or unenforceable, then that provision shall be deemed severable from these Terms and shall not affect the validity and enforceability of any remaining provisions of these Terms.
 
-We reserve the right to modify these Terms at any time by posting revised Terms on https://exercism.io. These Terms apply to all who access or use our Platform, registered or otherwise.
+We reserve the right to modify these Terms at any time by posting revised Terms on https://exercism.org. These Terms apply to all who access or use our Platform, registered or otherwise.
 
-Any personal information submitted to our Platform is subject to our [Privacy Policy](https://exercism.io/privacy).
+Any personal information submitted to our Platform is subject to our [Privacy Policy](https://exercism.org/docs/using/legal/privacy-policy).
```
Comment (iHiD) [inline pages/terms_of_service.md:43]: Same here.

**Comment by iHiD** [comment]
Thanks for this. I've made a few comments :)

**Comment by iHiD** [comment]
@peterloane Thanks. @ErikSchierboom This looks good to me. Could you check it pls.

File: pages/terms_of_service.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
 # Terms of Service
 
-Last Revised: April 2019
+Last Revised: September 2021
```
Comment (ErikSchierboom) [inline pages/terms_of_service.md:3]: Should this be November?

```suggestion
Last Revised: November 2021
```

**Review by ErikSchierboom — COMMENTED** [review]
One tiny nit.

File: pages/terms_of_service.md (hunk 3 above)
Comment (peterloane) [reply pages/terms_of_service.md:3]: @ErikSchierboom yes. Updated to November

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**